ci/build.gradle

```diff
@@ -54,6 +54,32 @@ task createDockerfile(type: Dockerfile, dependsOn: copyFiles) {
   }
 
   def branchName = getGitBranch()
+
+  user("root")
+
+  runCommand """\\
+                chmod 777 /envs && \\
+                for pythonVersion in ${pythonEnvironments}; \\
+                do \\
+                  apt-get install -y python\$pythonVersion && \\
+                  if [ \$pythonVersion = "3.8" ] ; then apt-get install -y python\$pythonVersion-distutils; fi \\
+                done
+                """
+
+  user("jenkins")
+
+  runCommand """\\
+                cd /envs && \\
+                pip install -U virtualenv && \\
+                for pythonVersion in ${pythonEnvironments}; \\
+                do \\
+                  virtualenv --python python\$pythonVersion sw_env_python\$pythonVersion && \\
+                  . /envs/sw_env_python\$pythonVersion/bin/activate && \\
+                  pip install -U setuptools && \\
+                  pip install twine; \\
```

Comment:
this library doesn't need to be installed also with -U?

Reply:
Actually, the `-U` parameter is not needed for `setuptools` since we are installing packages to newly created virtual environments. Will update, thank you!